flixel/graphics/FlxGraphic.hx

```diff
@@ -499,7 +499,8 @@ class FlxGraphic implements IFlxDestroyable
 		if (collection.type != null)
 		{
 			var collections:Array<Dynamic> = getFramesCollections(collection.type);
-			collections.push(collection);
+			if (collections.indexOf(collection) == -1)
```

Comment:
Should I add this to flixel? what does it improve?

Reply:
I want to say I made this change back when I was working on trying to resolve an issue with FlxAnimate. I'd definitely keep it as you don't want the list of collections to have duplicate references in it (as that can cause the reference counts to get messed up).